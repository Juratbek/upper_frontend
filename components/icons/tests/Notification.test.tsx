import { render, screen } from '@testing-library/react';

import { NotificationIcon } from '../Notification';

describe('Notification icon', () => {
  it('renders the icon', () => {
    render(<NotificationIcon />);
    const icon = screen.getByRole('icon');
    expect(icon).toMatchSnapshot();
  });
});
