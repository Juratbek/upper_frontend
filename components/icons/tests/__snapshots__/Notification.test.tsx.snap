// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Notification icon > renders the icon 1`] = `
<svg
  fill="none"
  height="24"
  role="icon"
  viewBox="0 0 24 24"
  width="24"
>
  <mask
    fill="white"
    id="path-2-inside-1_630_4298"
  >
    <path
      d="M4.5 10.5C4.5 6.35786 7.85786 3 12 3C16.1421 3 19.5 6.35786 19.5 10.5V17C19.5 17.5523 19.0523 18 18.5 18H5.5C4.94772 18 4.5 17.5523 4.5 17V10.5Z"
    />
  </mask>
  <path
    d="M4.5 10.5C4.5 6.35786 7.85786 3 12 3C16.1421 3 19.5 6.35786 19.5 10.5V17C19.5 17.5523 19.0523 18 18.5 18H5.5C4.94772 18 4.5 17.5523 4.5 17V10.5Z"
    mask="url(#path-2-inside-1_630_4298)"
    stroke="black"
    stroke-width="2.8"
  />
  <path
    d="M16 17C16 17.5253 15.8965 18.0454 15.6955 18.5307C15.4945 19.016 15.1999 19.457 14.8284 19.8284C14.457 20.1999 14.016 20.4945 13.5307 20.6955C13.0454 20.8965 12.5253 21 12 21C11.4747 21 10.9546 20.8965 10.4693 20.6955C9.98396 20.4945 9.54301 20.1999 9.17157 19.8284C8.80014 19.457 8.5055 19.016 8.30448 18.5307C8.10346 18.0454 8 17.5253 8 17L12 17H16Z"
    fill="black"
  />
  <path
    d="M7 18H17V19H7V18Z"
    fill="white"
  />
</svg>
`;
